src/kaocha/core_ext.clj

```diff
@@ -1,7 +1,14 @@
 (ns kaocha.core-ext
   "Core language extensions"
   (:refer-clojure :exclude [symbol])
-  (:import [java.util.regex Pattern]))
+  (:import [java.util.regex Pattern])
+  (:require  [kaocha.output :as output]
+            [slingshot.slingshot :refer [try+ throw+]]))
+
+;This ends up being the best place to put the version check to avoid duplication.
+(when-not (and (>= (:major *clojure-version*) 1) (>= (:minor *clojure-version*) 9))
```

Comment:
If Clojure ever gets a 2.0 this is going to break :)

Reply:
Ahh, right of course. Fixed! :)